xgcm/grid.py

```diff
@@ -1080,11 +1080,12 @@ def __init__(
         face_connections : dict
             Grid topology
         coords : dict, optional
-            Explicit specification of axis coordinates, e.g
+            Explicit specification of axis dimensions, e.g
             ``{'X': {'center': 'XC', 'left: 'XG'}}``.
             Each key should be the name of an axis. The value should be
```

Comment:
Add something like this? XC, XG are only understandable to MITgcm-adajacent people.

```
Keys should be one of `X`, `Y`, or `Z` and map to a dictionary mapping positions (`center`, `left`, `right`, `outer`, `inner`) to dimension names in the dataset (here, `XC` and `XG`).
```

Reply:
That is great!